filebeat/main.go

```diff
@@ -3,12 +3,9 @@ package main
 import (
 	"os"
 
-	"github.com/elastic/beats/filebeat/beater"
-	"github.com/elastic/beats/libbeat/beat"
+	"github.com/elastic/beats/filebeat/cmd"
```

Comment:
I promise this was already there: 

![image](https://cloud.githubusercontent.com/assets/299804/26672736/9ee4d8fa-46ba-11e7-8ace-58e687556e83.png)

:smile: